README.md

```diff
@@ -198,6 +200,8 @@ Note: items in bold are **required**.
 
 **`showMarkers`**: This controls whether markers are shown at the centerpoint of each chapter. If `true`, the map will display a default blue, inverted-teardrop icon.
 
+**`markerColor`**: If `showMarkers` is `true`, this property will override the default light blue marker color.
```

Comment:
Can the color be hex or rgb? Does it matter? You may want to provide that information to make sure people provide the correct format.